src/System.Private.ServiceModel/tests/Common/Infrastructure/xunit/WcfFactAttribute.cs

```diff
@@ -9,7 +9,8 @@
 namespace Infrastructure.Common
 {
     // TODO: Use the common tests projects instead of hardcoding System.ServiceModel.Primitives here.
-    [XunitTestCaseDiscoverer("Infrastructure.Common.WcfFactDiscoverer", "System.ServiceModel.Primitives.Tests")]
+    [XunitTestCaseDiscoverer("Infrastructure.Common.WcfFactDiscoverer", "Infrastructure.Common")]
+    //[XunitTestCaseDiscoverer("Infrastructure.Common.WcfFactDiscoverer", "System.ServiceModel.Primitives.Tests")]
```

Comment:
I think you should get rid of this line and the TODO--then this file will be unchanged by the BT2.0 effort.